modules/ipi-install-bmc-addressing.adoc

```diff
@@ -64,3 +64,62 @@ platform:
           password: <password>
           disableCertificateVerification: True
 ----
+[discrete]
+== Redfish APIs
+
+Several redfish API endpoints are called onto your BCM when using the bare-metal installer-provisioned infastructure.
+
+[IMPORTANT]
+====
+You need to ensure that your BMC supports all of the redfish APIs prior to installation.
+====
+
+**List of redfish APIs:**
+
+.PowerOn
+[source, terminal]
+----
+curl -u $USER:$PASS -X POST -H'Content-Type: application/json' -H'Accept: application/json' -d '{"Action": "Reset", "ResetType": "On"}' https://$SERVER/redfish/v1/Systems/$SystemID/Actions/ComputerSystem.Reset
+----
+
+.PowerOff
+[source, terminal]
+----
+curl -u $USER:$PASS -X POST -H'Content-Type: application/json' -H'Accept: application/json' -d '{"Action": "Reset", "ResetType": "ForceOff"}' https://$SERVER/redfish/v1/Systems/$SystemID/Actions/ComputerSystem.Reset
+----
+
+.Temporary boot
```

Comment:
Important disclaimer: I understand very little about APIs. So this may be a silly question :sweat_smile: 

Why are some of these things that look like object names (`BootSourceOverrideMode`) and some more normal looking phrases (`Temporary boot`)?

Reply:
I do know they are suppose to be phrases/descriptors, I believe I might have messed up the spacing here and `BootSourceOverrideMode` was meant to be like `Boot source OverrideMode`. I'll check with dev too